run_unittests.py

```diff
@@ -5029,6 +5029,27 @@ def test_wrap_git(self):
             self.build()
             self.run_tests()
 
+    def test_nostdlib(self):
```

Comment:
this should be in the `NativeFileTests` object, which has helper methods for writing machine files.

Reply:
It does both native and cross. tbh this test is trivial enough to not need any helper. You can fake cross build with empty cross file, and it cross build to the same platform.

Reply:
(was leaking the tmp file, there are other tests doing that... anyway, fixed this one)

Reply:
The helpers though don't used NamedTemprorary file which has all kinds of caveats on windows and we shouldn't be using.

Reply:
Well, if that doesn't work, no sure how our CI works then...
```
meson$ git grep "tempfile.TemporaryDirectory" | wc -l
30
```

That said, thinking about it I could write the file into builddir instead.

Reply:
At least use the `temp_filename` helper? I'd be happy with that. It doesn't have the caveats that NamedTemporaryFile does.

Reply:
Already updated to use file in build dir ;)